### DISCUSSION TITLE
language markup / css

### DISCUSSION THREAD

**Discussion by IwateKyle** [pocketbase/pocketbase#2164] [Q&A] [answered]
Hi, I added a block of javascript code to a post body using Pocketbase rich-text editor.  I was thrilled when I saw that the javascript code I inserted was color coded nicely. 

However, when I pulled that record from the database and inserted the body into my html page, the color formatting disappeared.  I'm assuming it's a css problem.  The code has a class of language-javascript as such...

`<pre class="language-javascript">....<pre>`

Is there a way I can export that code formatting?

**Comment by ganigeorgiev** [comment, marked as answer]
Based on the [TinyMCE Code Sample Plugin docs](https://www.tiny.cloud/docs/tinymce/6/codesample/#using-prismjs-on-your-web-page), you'll have to load [Prism.js](https://prismjs.com/) in your page in order to get the syntax highlighter to work.

You can use the [Prism download configurator](https://prismjs.com/download.html#themes=prism&languages=markup+css+clike+javascript) to export the syntax highlighted styles only for the languages you need in the desired color scheme and then load the generated js and css files:

```html
<link rel="stylesheet" type="text/css" href="prism.css">
<script src="prism.js"></script>
```

**Comment by IwateKyle** [comment]
thank you !!
